src/main/resources/i18n/messages.properties

```diff
@@ -514,7 +515,32 @@ UserDetailsPage.form.button.submit=Update User
 UserDetailsPage.createdDate=Created on {0}
 UserDetailsPage.modifiedDate=Modified on {0}
 UserDetailsPage.lastLogin=Last login on {0}
-UserDetailsPage.loading.message=Loading user details.
+
+UserSecurityPage.title=Security
+UserSecurityPage.alert.title=Missing Email Configuration
+UserSecurityPage.alert.description=Password reset is disabled, because IRIDA is not currently configured to send e-mails.
+
+UserChangePasswordForm.title=Change Password
+UserChangePasswordForm.alert.title=Password Policy
+UserChangePasswordForm.alert.description=Please choose a strong password that meets the following criteria:
+UserChangePasswordForm.alert.rule1=Must not be empty.
+UserChangePasswordForm.alert.rule2=Must be at least 8 characters long.
+UserChangePasswordForm.alert.rule3=Must contain at least 1 uppercase letter.
+UserChangePasswordForm.alert.rule4=Must contain at least 1 lowercase letter.
+UserChangePasswordForm.alert.rule5=Must contain at least 1 number.
+UserChangePasswordForm.alert.rule6=Must contain at least 1 special character.
```

Comment:
Add some example in parantheses of special characters...(!,$, etc)

Reply:
Please see 454982024ee7040bbcf647e055d5022cad6b14ec.